### ISSUE TITLE
Refactor opening times

### ISSUE THREAD

**Issue by c00k133** [Teknologforeningen/www.teknologforeningen.fi#45]
We should refactor how opening times are controlled on the page. Currently the implementation includes hard-coded strings, which has been a hell to moderate.

**Comment by c00k133** [comment]
Additionally we should refactor properly how times are updated. I do not know the actual times anymore, but this is the mail I received last year:

> Menyn på framsidan visar varje dag fr.o.m. kl 15.00 maträtterna för nästa dag trots att TF har öppet till 17.30 på onsdag. Menyn på framsidan borde alltså programmeras om så att övergången till nästa dag sker först kl 17.30 på onsdagar.

**Comment by VonLatvala** [comment]
Yes, this has been a problem from the get-go, since the summer/nonsummer hours have not been changing on predictable enough dates. Also as this is a pure frontend software, there really hasn’t been any other way to do this than editing the php files themselves. I don’t think Täffä has any API which could be queried for opening hours, but if they do have one, using it would be a solution to this _mess_.

**Comment by tlangens** [comment]
They do have an API based on some DB that I believe they manage via some MS Access application, that I recently ported to Flask: https://github.com/Teknologforeningen/dagsenapi-flask

One could look into adding opening hours to this but then again it might make sense to just rewrite it all to something more modern.
